.github/workflows/frontend.yml

```diff
@@ -2,22 +2,36 @@ name: Frontend CI
 
 on:
   push:
-    branches: [main]
+    branches: [ main ]
     paths:
       - '.github/workflows/frontend.yml'
-      - 'frontend/**/*'
-      - '!frontend/.simple-git-hooks.json'
-      - '!frontend/.lintstagedrc.json'
+      - 'typescript/**/*'
```

Comment:
There is no `typescript` directory. Should this be `ts/**/*`? Same issue also below under `pull_request` section.